flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/util/collections/ByteHashSet.java

```diff
@@ -43,109 +39,18 @@ public ByteHashSet() {
 	}
 
 	public boolean add(final byte k) {
-		if (k == 0) {
-			if (this.containsZero) {
-				return false;
-			}
-
-			this.containsZero = true;
+		if (used[k - min]) {
```

Comment:
Just simplify it: `return !used[k - min] && (used[k - min] = true);`

Reply:
Thanks for the good suggestion. Your code is definitely more efficient.